fe/fe-core/src/main/java/org/apache/doris/datasource/iceberg/action/IcebergRewriteDataFilesAction.java

```diff
@@ -0,0 +1,86 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.datasource.iceberg.action;
+
+import org.apache.doris.catalog.TableIf;
+import org.apache.doris.common.DdlException;
+import org.apache.doris.common.UserException;
+import org.apache.doris.datasource.iceberg.IcebergExternalTable;
+import org.apache.doris.nereids.trees.expressions.Expression;
+import org.apache.doris.nereids.trees.plans.commands.info.PartitionNamesInfo;
+
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Iceberg rewrite data files action implementation.
+ * This action rewrites data files in Iceberg tables to optimize file sizes
+ * and improve query performance.
+ */
+public class IcebergRewriteDataFilesAction extends BaseIcebergAction {
+    public static final String TARGET_FILE_SIZE = "target_file_size";
+    public static final String MIN_INPUT_FILES = "min_input_files";
```

Comment:
No `MIN_FILE_SIZE_MB ` and `MAX_FILE_SIZE_MB `?
See: https://www.dremio.com/blog/optimizing-iceberg-tables/